opentelemetry/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - *Breaking* Change return type of `opentelemetry::global::set_tracer_provider` to Unit to align with metrics counterpart
 - Add `get_all` method to `opentelemetry::propagation::Extractor` to return all values of the given propagation key and provide a default implementation.
+- Add support for max span attribute value length; Applies to `Value::String` and `Array::String` only.
```

Comment:
Attribute length enforcements are not part of `opentelemetry` crate. Its part of `opentelemetry_sdk` crate, so changelog should be in that crate.